client/src/App.tsx

```diff
@@ -1,16 +1,19 @@
 import React from 'react';
 import ThemeContextProvider from 'context/theme/provider';
 import ThemeProvider from 'theme/themeProvider';
-import Page from 'components/Page';
-import ThemeToggle from 'components/buttons/ThemeToggle';
+import { BrowserRouter, Route } from 'react-router-dom';
+import Home from 'pages/Home';
+import Membership from 'pages/Membership';
+import { ROUTES } from 'utils/constants';
 
 function App() {
   return (
     <ThemeContextProvider>
       <ThemeProvider>
-        <Page>
-          <ThemeToggle />
-        </Page>
+        <BrowserRouter>
+          <Route exact path={ROUTES.HOME} component={Home} />
+          <Route path={ROUTES.MEMBERSHIP} component={Membership} />
```

Comment:
Should these page `Route`s be in a `Switch` so that only one can ever be rendered?